// bootstrap-styled/lib/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "theme", {
  enumerable: true,
  get: function get() {
    return _theme.default;
  }
});
Object.defineProperty(exports, "makeTheme", {
  enumerable: true,
  get: function get() {
    return _theme.makeTheme;
  }
});
Object.defineProperty(exports, "createMakeTheme", {
  enumerable: true,
  get: function get() {
    return _utils.default;
  }
});
Object.defineProperty(exports, "makeScopedTheme", {
  enumerable: true,
  get: function get() {
    return _utils.makeScopedTheme;
  }
});
Object.defineProperty(exports, "toMakeTheme", {
  enumerable: true,
  get: function get() {
    return _utils.toMakeTheme;
  }
});
Object.defineProperty(exports, "getGlobalStyles", {
  enumerable: true,
  get: function get() {
    return _reboot.getGlobalStyles;
  }
});
Object.defineProperty(exports, "getGlobalStyleNoBootstrapProvider", {
  enumerable: true,
  get: function get() {
    return _reboot.getGlobalStyleNoBootstrapProvider;
  }
});

var _theme = _interopRequireWildcard(require("./theme"));

var _utils = _interopRequireWildcard(require("./utils"));

var _reboot = require("@bootstrap-styled/css-utils/lib/reboot");

function _getRequireWildcardCache() { if (typeof WeakMap !== "function") return null; var cache = new WeakMap(); _getRequireWildcardCache = function _getRequireWildcardCache() { return cache; }; return cache; }

function _interopRequireWildcard(obj) { if (obj && obj.__esModule) { return obj; } var cache = _getRequireWildcardCache(); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; if (obj != null) { var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } } newObj.default = obj; if (cache) { cache.set(obj, newObj); } return newObj; }
