// bootstrap-styled/lib/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "theme", {
  enumerable: true,
  get: function get() {
    return _theme.default;
  }
});
Object.defineProperty(exports, "makeTheme", {
  enumerable: true,
  get: function get() {
    return _theme.makeTheme;
  }
});
Object.defineProperty(exports, "createMakeTheme", {
  enumerable: true,
  get: function get() {
    return _utils.default;
  }
});
Object.defineProperty(exports, "makeScopedTheme", {
  enumerable: true,
  get: function get() {
    return _utils.makeScopedTheme;
  }
});
Object.defineProperty(exports, "toMakeTheme", {
  enumerable: true,
  get: function get() {
    return _utils.toMakeTheme;
  }
});
Object.defineProperty(exports, "getGlobalStyles", {
  enumerable: true,
  get: function get() {
    return _reboot.getGlobalStyles;
  }
});
Object.defineProperty(exports, "getGlobalStyleNoBootstrapProvider", {
  enumerable: true,
  get: function get() {
    return _reboot.getGlobalStyleNoBootstrapProvider;
  }
});

var _theme = _interopRequireWildcard(require("./theme"));

var _utils = _interopRequireWildcard(require("./utils"));

var _reboot = require("@bootstrap-styled/css-utils/lib/reboot");

function _interopRequireWildcard(obj) { if (obj && obj.__esModule) { return obj; } else { var newObj = {}; if (obj != null) { for (var key in obj) { if (Object.prototype.hasOwnProperty.call(obj, key)) { var desc = Object.defineProperty && Object.getOwnPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : {}; if (desc.get || desc.set) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } } newObj.default = obj; return newObj; } }
